third_party/tpch-dbgen/tpch_dbgen.h

```diff
@@ -50,4 +50,5 @@ extern distribution vp;
 extern distribution grammar;
 
 void dbgen_reset_seeds();
+void dbgen_init_scale_factor(float scale_factor);
```

Comment:
Did you diff our dbgen and the original one? Otherwise, I'd do it and check if anything sticks out.

Reply:
I checked it manually, seems fine. Left more `// HYRISE: We changed this` comments

https://gist.github.com/mrzzzrm/82ac2721e87eb5a6645a5482c0ab7f7c